packages/opensrp-plans/src/ducks/configs/settings.ts

```diff
@@ -0,0 +1,193 @@
+import {
```

Comment:
These seem like types, they can go into a plans-global types file, i.e.  would not think of them as configuration stuff, what do you think